lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,14 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+
+    tokens = text.lower()
+    for symbol in text:
```

Comment:
Le't create a list and add to it if symbol is alpha? It should be easier

Reply:
Probably fixed..

Reply:
Are you sure?

Reply:
No :(